WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingTagsFragment.kt

```diff
@@ -0,0 +1,27 @@
+package org.wordpress.android.ui.posts
+
+import android.content.Context
+import android.os.Bundle
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.fluxc.model.SiteModel
+
+class PrepublishingTagsFragment : TagsFragment() {
```

Comment:
I know we aren't a big fan of inheritance. I just utilized it as I thought it was the simplest way to reuse the existing `TagsFragment` functionality. More details on this here. https://github.com/wordpress-mobile/WordPress-Android/pull/11644

Reply:
I think this is a valid use of inheritance. Both the screens display the same data and serve the same purpose (tag selection). AFAIU the only difference between them is that they have a bit different UI, right?

Reply:
Yes, exactly. I am just happy the approach is warranted 🙏